samples/nrf9160/lte_ble_gateway/CMakeLists.txt

```diff
@@ -20,7 +20,9 @@ set(spm_CONF_FILE
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(lte-ble-gateway)
 
+# NORDIC SDK APP START
 target_sources(app PRIVATE src/main.c)
+# NORDIC SDK APP END
 zephyr_include_directories(src)
 
 include_directories(src/orientation_detector)
```

Comment:
This include is not part of the SDK APP?

Reply:
No, cause it is not limited to the target `app`

SES only supports the following CMake functions in its integration to the IDE.
- `target_sources`
- `target_compile_definitions`
- `target_include_directories`
- `target_compile_options`